api/types.proto

```diff
@@ -521,6 +536,8 @@ message PortConfig {
 	// within the available range. If not specified, an available
 	// port is automatically assigned.
 	uint32 published_port = 4;
+
+	Exposition exposition = 5;
```

Comment:
@aaronlehmann @mrjana @aluzzardi @mavenugo @mstanleyjones I really hate this terminology. Calling in the semantic adaptationists.